playbooks/roles/edxapp/templates/edx/app/edxapp/worker.sh.j2

```diff
@@ -7,6 +7,18 @@ source {{ edxapp_app_dir }}/edxapp_env
 {% if COMMON_ENABLE_NEWRELIC_APP %}
 {% set executable = edxapp_venv_bin + '/newrelic-admin run-program ' + edxapp_venv_bin + '/celery' %}
```

Comment:
We'll need to have something more like this:
```suggestion
{% set executable = edxapp_venv_bin + '/newrelic-admin run-program ' + executable %}
```
...and set the base value for `executable` before the conditional block, as well as some other rearrangement farther down. Otherwise this would stop working when we disabled NR. (I had to make that change to the lms and cms files previously.)